apps/api/src/controllers/player.ts

```diff
@@ -2,15 +2,17 @@ import { Request, Response } from 'express';
 
 export default {
   GET_fetchAllPlayers: async (req: Request, res: Response) => {
-    const players = await req.ctx.modelFactory.player.fetchAll(req.ctx);
+    const { page, pageSize } = req.query;
+    const pageNumber = parseInt(page as string) || 1;
```

Comment:
```suggestion
    const pageNumber = max(parseInt(page as string) || 1, 1);
```
prevent negative page number